tracer/samples/AzureFunctionsWithAgentLessLogging/FunctionWithLogging/run.ps1

```diff
@@ -0,0 +1,133 @@
+<#
+.SYNOPSIS
+    A script that allows the user to run C# sample applications to demonstrate
+    the setup and usage of Datadog's .NET Tracer log collection and log trace ID injection in a function
+.PARAMETER AppDirectory
+    The application directory - defaults to the location of this script and need not be set
+    by the user unless samples are moved around.
+.PARAMETER ApiKey
+    The Datadog API key (only required when -Agentless is set)
+.EXAMPLE
+    pwsh BuildAndRunSample.ps1 -ApiKey YOUR_API_KEY_HERE
+#>
+
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory = $false)]
+    [System.String]
+    $AppDirectory, # only necessary if the script is removed from its current directory layout
+
+    [Parameter(Mandatory = $false)]
```

Comment:
This one might actually be mandatory? Maybe not though if they have it set outside?